R/methods.R

```diff
@@ -141,3 +141,28 @@ as.data.frame.lints <- function(x, row.names = NULL, optional = FALSE, ...) {
   attributes(x) <- attrs
   x
 }
+
+#' @export
+summary.lints <- function(object, ...) {
+  filenames <- vapply(object, `[[`, character(1), "filename")
+  types <- factor(vapply(object, `[[`, character(1), "type"),
+    levels = c("style", "warning", "error"))
+  tbl <- table(filenames, types)
+  filenames <- rownames(tbl)
```

Comment:
change this line to
```r
filenames <- rownames(tbl) %||% character()
```

Then you can remove the entire conditional at https://github.com/jimhester/lintr/pull/262/files#diff-a0a9d3c78e9f113319e5e069fb2697ecR155

Reply:
Now fixed.